umibukela/static/stylesheets/site.scss

```diff
@@ -610,10 +585,10 @@ footer.footer{
 		padding: 5px;
 		border-radius: 3px;
 		h1, h2 { margin: 0; color: $white ;fa}
```

Comment:
Not sure either. It may be something with font-awesome. Though it almost seems more like typo.